test/fitnesse/responders/testHistory/PurgeHistoryResponderTest.java

```diff
@@ -68,14 +70,23 @@ public void shouldMakeErrorResponseWhenItGetsInvalidTypeForNumberOfDays() throws
     request.addInput("days", "bob");
     Response response = responder.makeResponse(context, request);
     assertEquals(400, response.getStatus());
-
+  }
+  
+  @Test
+  public void shouldDeleteHistoryFromRequestForWikiPath() throws Exception {
```

Comment:
Can you also add a test case for when the `purgeGlobal` parameter is not set?

And I think an assertion is needed that shows the correct path is supplied to the `deleteTestHistoryOlderThanDays` method